_episodes/09-pull-requests.md

```diff
@@ -155,28 +234,48 @@ Writing objects: 100% (4/4), 783 bytes | 0 bytes/s, done.
 Total 4 (delta 3), reused 0 (delta 0)
 remote: Resolving deltas: 100% (3/3), completed with 3 local objects.
 To https://github.com/USERNAME/countries.git
-   2037539..79a312a  master -> master
+   2037539..79a312a  addFrance -> addFrance
 ~~~
 {: .output}
 
 Next go to your forked github version of the repo and reload the page.
-You should see the changes in your version of the repo.
+You won't see the new file added in the list of files but you will see 
+that you recently pushed a new branch to the repository.
+
+![](../fig/github_screenshot_origin_master_addFrance.png)
+
+If you wish to view your new branch you can click on the 'Branch' drop down menu
+and select that branch.
+
+![](../fig/github_screenshot_switch_github_branch.png)
 
-![](../fig/github_screenshot_origin_wFrance.png)
+Then you should be able to view the files and commits in that branch.
+
+![](../fig/github_screenshot_origin_master_addFrance.png)
+
+Github already suspects that we are going to want to make a pull request so we can click
+the 'Compare & pull request' button to start a new pull request.
 
 Then click the 'New pull request' button.
 
 ![](../fig/github_screenshot_makingPR1.png)
 
-The base fork should be the upstream version and then the head fork should be your version.
-Then click the 'create pull request button'.
+The base fork should be the upstream/authoritative version's master branch and then 
```

Comment:
Yeah, I think the wording for these two branches when making a PR is really bad.  It confuses even seasoned git users which is the head and which is the base.